lib/chat-client/index.js

```diff
@@ -29,29 +29,41 @@ class Chat extends ClientBase {
    * @param options - An object of options that can be passed to the method.
    * @returns - An array of chat conversations. If there are no conversations, the array is empty.
    * @example
-   * const chatConversations = bot.chat.list({unreadOnly: true})
+   * bot.chat.list({unreadOnly: true}).then(chatConversations => console.log(chatConversations))
    */
-  async list(options: ChatListOptions): Promise<ChatConversation[]> {
+  async list(options?: ChatListOptions): Promise<ChatConversation[]> {
     await this._guardInitialized()
     const res = await this._runApiCommand({apiName: 'chat', method: 'list', options})
-    return res ? res.conversations || [] : []
+    if (!res) {
+      throw new Error('Keybase chat send returned nothing.')
+    }
+    return res.conversations || []
   }
 
   /**
    * Reads the messages in a channel. You can read with or without marking as read.
    * @memberof Chat
    * @param channel - The chat channel to read messages in.
    * @param options - An object of options that can be passed to the method.
+   * @returns - A summary of data about a message, including who send it, when, the content of the message, etc. If there are no messages in your channel, then an error is thrown.
```

Comment:
shouldn't an empty chat just reply with an empty array? instead of throwing an error? it seems to me asking for a list of something and that list being empty doesn't imply an error, just a successful empty list.

Reply:
That was what the client was doing, so I just passed that behavior on. All for making it an empty array though, that makes more sense.

Reply:
ok, I think I know why this is happening. let's leave it in core's hands too.